### DISCUSSION TITLE
MPD executable 2MB > 3 MB since 6 September

### DISCUSSION THREAD

**Discussion by skidoo23** [MusicPlayerDaemon/MPD#1866] [Q&A] [answered]
Is it just me? My current MPD executable (compiled from master, on Debian testing x64) grew from more or less 2 meg to 3 meg. Growth happened with commits from 06-sep-2023.

**Comment by MaxKellermann** [comment]
True, 44beae519d1966ebf96d53f858f4ea15932d0849 is a heavy increase because the switch to a doubly-linked list adds lots of initialized data; the singly-linked list could just be initialized with nullptr, i.e. it lived in .bss and not .data.

**Reply by MaxKellermann** [reply, marked as answer]
Back to normal size by commit 3e09f95b40d2f0fe4ea8bf59aba6406274b0dfcd

**Reply by skidoo23** [reply]
Thx!
